Microsoft.Toolkit.Uwp.UI.Controls/TileControl/TileControl.cs

```diff
@@ -453,90 +453,73 @@ private void CreateModuloExpression(ScrollViewer scrollViewer, double imageWidth
-                    offsetXNode == 0,
-                    0,
-                    ExpressionFunctions.Conditional(
-                        offsetXNode < 0,
-                        -(ExpressionFunctions.Abs(offsetXNode - (ExpressionFunctions.Ceil(offsetXNode / imageWidthNode) * imageWidthNode)) % imageWidthNode),
-                        -(imageWidthNode - (offsetXNode % imageWidthNode))));
-
-                expressionYVal = ExpressionFunctions.Conditional(
-                    offsetYNode == 0,
-                    0,
-                    ExpressionFunctions.Conditional(
-                        offsetYNode < 0,
-                        -(ExpressionFunctions.Abs(offsetYNode - (ExpressionFunctions.Ceil(offsetYNode / imageHeightNode) * imageHeightNode)) % imageHeightNode),
-                        -(imageHeightNode - (offsetYNode % imageHeightNode))));
+                expressionXVal = Thing("Ceil(" + offsetXParam + ")", imageHeightParam);
+
+                expressionYVal = Thing("Ceil(" + offsetYParam + ")", imageWidthParam);
             }
             else
             {
                 // expressions are created to simulate a positive and negative modulo with the size of the image and the offset and the ScrollViewer offset (Translation)
                 var scrollProperties = ElementCompositionPreview.GetScrollViewerManipulationPropertySet(scrollViewer);
-                var scrollPropSet = scrollProperties.GetSpecializedReference<ManipulationPropertySetReferenceNode>();
-
-                var speed = propertySetNodeModulo.GetScalarProperty(speedParam);
-                var xCommon = ExpressionFunctions.Ceil((scrollPropSet.Translation.X * speed) + propertySetNodeModulo.GetScalarProperty(offsetXParam));
-                expressionXVal = ExpressionFunctions.Conditional(
-                    xCommon == 0,
-                    0,
-                    ExpressionFunctions.Conditional(
-                        xCommon < 0,
-                        -(ExpressionFunctions.Abs(xCommon - (ExpressionFunctions.Ceil(xCommon / imageWidthNode) * imageWidthNode)) % imageWidthNode),
-                        -(imageWidthNode - (xCommon % imageWidthNode))));
-
-                var yCommon = ExpressionFunctions.Ceil((scrollPropSet.Translation.Y * speed) + propertySetNodeModulo.GetScalarProperty(offsetYParam));
-                expressionYVal = ExpressionFunctions.Conditional(
-                    yCommon == 0,
-                    0,
-                    ExpressionFunctions.Conditional(
-                        yCommon < 0,
-                        -(ExpressionFunctions.Abs(yCommon - (ExpressionFunctions.Ceil(yCommon / imageHeightNode) * imageHeightNode)) % imageHeightNode),
-                        -(imageHeightNode - (yCommon % imageHeightNode))));
+                expressionX.SetReferenceParameter("s", scrollProperties);
+                expressionY.SetReferenceParameter("s", scrollProperties);
```

Comment:
Should we use `scroll` vs. `s` here?

Reply:
Sure we can. My latest commits makes it so that the string is just fed into the API and only ref'ing it from a const in the code, so its either way IMO. Just one char might be more performant since we have a const for it I see nothing wrong with the string being non descriptive.